src/common/Card/Reminder/Reminder.tsx

```diff
@@ -4,12 +4,17 @@ import { ReminderProps } from "../interfaces";
 import * as UI from "./style";
 import PropTypes from "prop-types";
 
+// Time constants for clarity
+const ONE_DAY_MINUTES = 1440;
+const FIFTEEN_MIN_MS = 15 * 60000;
```

Comment:
Rename to more generic constants (constants that just dont restate what it is). Also use the same time unit for both constants (ms, secs, mins, etc)

Ex
```javascript
const TIME_UNTIL_HIDE_BADGE_MS;
const TIME_BEFORE_SHOW_BADGE_MS; 
```